Gemfile.lock

```diff
@@ -809,7 +809,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.6.6p146
+   ruby 2.6.5p114
```

Comment:
beware this is not correct, current Ruby version should be 2.6.6, better remove this change

Reply:
Ooops, my bad!